.github/workflows/psp-ci.yml

```diff
@@ -0,0 +1,44 @@
+name: CI PSP
+
+on:
+  push:
+  pull_request:
+  repository_dispatch:
+    types: [run_build]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: pspdev/pspdev:latest
+
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Prepare Distribution
+      run: |
+        ./configure --host=psp --disable-debug --enable-plugins --default-dynamic --enable-release
+        make distclean
+
+    - name: Compile ScummVM
+      run: |
+        ./configure --host=psp --disable-debug --enable-plugins --default-dynamic --enable-release
+        make -j$(getconf _NPROCESSORS_ONLN)
+
+    - name: Prepare Release
+      run: |
+        export VERSION=`make print-distversion`
+        export DISTS=`make print-dists`
+        mkdir scummvm-$VERSION
+        cp -r $DISTS EBOOT.PBP plugins scummvm-$VERSION
+        mkdir scummvm-$VERSION/kbd
+        cp -r backends/platform/psp/kbd/*.png scummvm-$VERSION/kbd
+
+    - name: Get Version
+      id: slug
+      run: echo "::set-output name=version::$(make print-distversion)"
+    
+    - uses: actions/upload-artifact@v2
```

Comment:
I would maybe limit this to tags - the problem is that once we do that for every push we might very soon run out of max space and the pipelines would fail.

Reply:
I still don't understand that, in theory there are unlimited space for the public repositories (the only limitation is with the cache that just keep the latest 10GB)

https://github.com/pricing?gclid=Cj0KCQjw4PKTBhD8ARIsAHChzRLnOeuLsYVkHuckA3_3C9BAkot40uhjdhn5-pdVLlOPCbQW1dxbdQMaAikGEALw_wcB#compare-features

Reply:
I still don't understand that, in theory there are unlimited space for the public repositories (the only limitation is with the cache that just keep the latest 10GB)

https://github.com/pricing?gclid=Cj0KCQjw4PKTBhD8ARIsAHChzRLnOeuLsYVkHuckA3_3C9BAkot40uhjdhn5-pdVLlOPCbQW1dxbdQMaAikGEALw_wcB#compare-features